docs/job-scheduling.md

```diff
@@ -54,6 +54,10 @@ Resource allocation can be configured as follows, based on the cluster type:
   (`spark.executor.memory` configuration property) and `--executor-cores` (`spark.executor.cores` configuration
   property) control the resources per executor. For more information, see the
   [YARN Spark Properties](running-on-yarn.html#spark-properties).
+* **K8s:** The same as the situation with Yarn, please refer to the description of Yarn above. Furthermore,
+  Spark on K8s offers higher priority versions of spark.kubernetes.executor.limit.cores and
+  spark.kubernetes.executor.request.cores than spark.executor.cores. For more information, see the
+  [K8s Spark Properties](running-on-kubernetes.html#spark-properties).
```

Comment:
spark.kubernetes.executor.limit.cores and
  spark.kubernetes.executor.request.cores than spark.executor.cores -> `spark.kubernetes.executor.limit.cores` and
  `spark.kubernetes.executor.request.cores` than `spark.executor.cores`